styleguide/template.md

```diff
@@ -320,6 +320,7 @@ Use three backticks (\`\`\`) + a language ID to apply language-specific color co
 |Swift|swift|
 |TypeScript|typescript|
 |VB|vb|
```

Comment:
```suggestion
|Visual Basic|vb|
```

Reply:
This column says what would show in the live docs, the visual basic code in the live docs appears as `VB`, so I think it should be kept as `VB` not `Visual Basic`.

![image](https://user-images.githubusercontent.com/31348972/65626828-66a72d80-dfce-11e9-8f41-0f5d748c2ead.png)